lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -701,16 +697,13 @@ bool WebsocketTlsTPM::connect() {
 }
 
 void WebsocketTlsTPM::reconnect(long delay) {
-    EVLOG_info << "Attempting TLS TPM reconnect with delay:" << delay;
```

Comment:
I would like this to be kept.

Reply:
We print the exact same thing on line 716. If you think it is better I can move remove the line on 716 and leave this one instead?

Reply:
I've simply missed that one sorry :). I think this would be better, because if we have a crash or anything in the 'close' this would be the last message printed and would give us a better clue on where the problem might be.

Reply:
I will move it to the middle then since it would be a bit strange to print `reconnecting` and immediately after that print `not reconnecting`.